XVim/XVimReplaceEvaluator.m

```diff
@@ -0,0 +1,95 @@
+//
+//  XVimReplaceEvaluator.m
+//  XVim
+//
+//  Created by Martin Conte Mac Donell on 12/14/14.
+//
+
+#import "XVimReplaceEvaluator.h"
+#import "XVimWindow.h"
+
+@interface XVimInsertEvaluator()
+
+@property (nonatomic) NSRange startRange;
+@property (nonatomic) BOOL movementKeyPressed;
+
+- (NSString*)insertedText;
+- (BOOL)windowShouldReceive:(SEL)keySelector;
+
+@end
+
+@implementation XVimReplaceEvaluator {
+    BOOL _oneCharMode;
```

Comment:
Can you just make this a property instead? Seems like these not being properties is left over from supporting earlier versions of stuff.